packages/server/src/environment.js

```diff
@@ -78,6 +78,7 @@ module.exports = {
   ALLOW_DEV_AUTOMATIONS: process.env.ALLOW_DEV_AUTOMATIONS,
   DISABLE_THREADING: process.env.DISABLE_THREADING,
   SQL_MAX_ROWS: process.env.SQL_MAX_ROWS,
+  TEMPLATE_REPOSITORY: process.env.TEMPLATE_REPOSITORY,
```

Comment:
something we want to promote more in future is default undefined values in this file directly so it's clear to see the state of the env in one place. This could become
```js
TEMPLATE_REPOSITORY: process.env.TEMPLATE_REPOSITORY || "app"
```
and then the var can just be referenced in the consumer